config/auth_azure_cli_test.go

```diff
@@ -1,4 +1,4 @@
-package config
+package config_test
```

Comment:
this will lead to code coverage not showing up for this package. check `make coverage`, please move tests back to `config` package

Reply:
make coverage shows that config package is covered by unit tests, including auth_metadata_server.
<img width="1485" alt="Screenshot 2023-06-15 at 10 51 05" src="https://github.com/databricks/databricks-sdk-go/assets/1850319/4c30b7e6-c4c1-4c39-88ba-87d0dedaee14">
<img width="599" alt="Screenshot 2023-06-15 at 10 50 56" src="https://github.com/databricks/databricks-sdk-go/assets/1850319/a9b973e7-7ca1-455f-b09d-9e930c4d8cea">